lib/SILGen/SILGenExpr.cpp

```diff
@@ -3782,7 +3783,69 @@ visitMagicIdentifierLiteralExpr(MagicIdentifierLiteralExpr *E, SGFContext C) {
   llvm_unreachable("Unhandled MagicIdentifierLiteralExpr in switch.");
 }
 
-RValue RValueEmitter::visitCollectionExpr(CollectionExpr *E, SGFContext C) {
+RValue RValueEmitter::visitArrayExpr(ArrayExpr *E, SGFContext C) {
+  FormalEvaluationScope Scope(SGF);
+
+  auto loc = SILLocation(E);
+  auto init = E->getInitializer();
+  auto *decl = dyn_cast<AbstractFunctionDecl>(init.getDecl());
+
+  auto elementType = E->getElementType();
+
+  auto argLabels = decl->getFullName().getArgumentNames();
+
+  llvm::SmallVector<CleanupHandle, 5> cleanups;
```

Comment:
SmallVector is injected into the Swift namespace. You can get rid of llvm:: here.